TabulateSmarterTestPackage/Program.cs

```diff
@@ -291,20 +291,20 @@ private static void TabulateResults(List<TestSpecificationProcessor> processors,
                     ReportingUtility.CrossProcessor.TestPackages.Values.Where(x => x.Count > 1);
                 foreach (var packageSets in crossTabulatedPackages)
                 {
-                    var adminPackage = packageSets.FirstOrDefault(x => x.PackageType == PackageType.Administration);
-                    if (adminPackage != null)
-                    {
-                        tabulator.TabulateResults(new List<TestSpecificationProcessor> {adminPackage},
-                            ReportingUtility.CrossProcessor.Errors[adminPackage.GetUniqueId()].Cast<ProcessingError>()
-                                .Where(x => x.PackageType == PackageType.Administration).ToList());
-                        continue;
-                    }
                     var scoringPackage = packageSets.FirstOrDefault(x => x.PackageType == PackageType.Scoring);
                     if (scoringPackage != null)
                     {
-                        tabulator.TabulateResults(new List<TestSpecificationProcessor> {scoringPackage},
+                        tabulator.TabulateResults(new List<TestSpecificationProcessor> { scoringPackage },
```

Comment:
Can you add a comment block above this that describes why it's okay to prioritize using the scoring package?  Basically what you described to me about the adminSegment not being needed in the tabulator.